asv/plugins/git.py

```diff
@@ -98,7 +98,7 @@ def get_date(self, hash):
     def get_hashes_from_range(self, range_spec):
         args = ['log', '--quiet', '--first-parent', '--format=format:%H']
         if range_spec != "":
-            args += [range_spec]
+            args += range_spec.split()
```

Comment:
Indeed, git rev-list indeed appears to not accept single arg spec..
("hash1 ^hash2" and so forth). There's a single-commit notation too
"hash1^!", though.